<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="FileButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QComboBox" name="COM_comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Bound_comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CheckBit_comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DataBit_comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="StopBit_comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenCom_pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FilePath_lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SendFile_pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
